### ISSUE TITLE
Bug: Tildes/Accent marks on task's comments don't work

### ISSUE THREAD

**Issue by cyphersprojekt** [mattermost/focalboard#2273]
## Steps to reproduce the behavior

1. Go to any task you've created / create a new one
2. Select  add a comment
3. Write anything that contains a tilde/accent mark (á, é, í, ó, ú)
4. That character disappears and the pointer goes back to the start of the text

## Expected behavior
Should be able to write with accent marks.
This issue doesn't happen on other text-boxes, like the description of the task you've created or its title

## Edition and Platform

 - Edition: Self hosted with the official docker image
 - Version: Latest
 - Browser and OS: Mozilla 96.0.3 Ubuntu 21.10

* Sev/3: Issue affects minor, non-critical functionality

**Comment by wuwinson** [comment]
Need to test extended character sets.

**Comment by wuwinson** [comment]
@chenilim @sbishel @ogi-m Tested [diacritic characters](https://en.wikipedia.org/wiki/Diacritic) on the properties, comments, and description section of [this card](https://community-daily.mattermost.com/boards/workspace/qgsck6cts3fwpqwyjiupjm5cde/845b05d4-3de9-4ffe-8b4e-356f67154910/c27424ff-ec94-4486-8772-ed1aef2e617c/ca6orceqqxffx7fbm9gchgkjmaw) and did not have any problems typing or copy/pasting these characters on any of the fields with the exception of asian characters within the **comments** section. Was able to copy/paste asian characters into the comments section, but was not able to type asian characters via the keyboard. Tiles/accent marks from latin characters worked fine from copy/paste and typing in all fields.

Tested on Boards Plugin on Community, Chrome 96 on MacOS.

In summary, unable to repro this bug #2273, but was able to repro #2343.

**Comment by chenilim** [comment]
@ogi-m, can you help double-check this on FireFox? It may be specific to that browser. Thanks.

**Comment by ogi-m** [comment]
@chenilim Couldn't repro in Chrome or Firefox on either the plugin or personal server.
